docs/REST_API.md

```diff
@@ -449,7 +449,7 @@ _**Example Response**_
 }
 ```
 **GET /v1/tasks/:id/watch**:
-Watch a task activity stream given a task ID
+Watch a task activity stream given a task ID. Watch is an event stream sent overa long running HTTP connection
```

Comment:
Many other sections like this do not have periods at the end. But because this one has two sentences maybe it should end with a period. I'll add that.